### DISCUSSION TITLE
Ask for Network finding precisions

### DISCUSSION THREAD

**Discussion by Domos2** [Yogui79/IntexPureSpa#37] [Q&A]
Hi there,

Great work congratulations !
I just want to knowsome things... I've found my channel, and I'll begin to search the network but I have questions...

How can I see that the right network has been found ? I will not be beside the esp all the time ... is the code script halting when right channel found ? Is something special written on screen or something like this ?
Yogui wrote that we can found it written in the log file but can we read this file during the scan without issue for the recording ? and last question how is it indicated that those channel is the right one ? 

All these questions because I'll have to let my laptop beside the spa outside so I just need some precisions before doing that.

I think that like for channel scan it is better to let the remote command on the pump base 5more signals per second if I remember well...)

Thanks for answering guys !

Best regards

**Comment by Yogui79** [comment]
sorry for my late answhere chaneel is printed on serial consol when founded
